hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -389,6 +392,7 @@ public OMResponse handleReadRequest(OMRequest request) {
   @Override
   public OMClientResponse handleWriteRequest(OMRequest omRequest,
       long transactionLogIndex) throws IOException {
+    injectPause();
```

Comment:
I think we can pause execution controlling double buffer also, using acquireUnFlushedTransactions() to acquire resource so that applyTransaction will wait till resource is available.

Reply:
This is for test. For the newly added test, one pause here is enough.